tests/debugger/test_debugger.py

```diff
@@ -35,13 +39,32 @@ def get_debugger_map():
 
                 if "diagnostics" in debugger:
                     probe_id = debugger["diagnostics"]["probeId"]
-                    hash["probes"][probe_id] = debugger["diagnostics"]
-
-                if "snapshot" in debugger:
+                    log_hash["probes"][probe_id] = debugger["diagnostics"]
+                elif "snapshot" in debugger:
                     probe_id = debugger["snapshot"]["probe"]["id"]
-                    hash["snapshots"][probe_id] = debugger["snapshot"]
+                    log_hash["snapshots"][probe_id] = debugger["snapshot"]
+
+    return log_hash
+
+
+def get_debugger_tracer_map():
+    agent_logs_endpoint_requests = list(interfaces.agent.get_data(path_filters="/api/v0.2/traces"))
+    span_hash = {}
+    for request in agent_logs_endpoint_requests:
+        content = request["request"]["content"]
+        if content is not None:
+            for payload in content["tracerPayloads"]:
+                for chunk in payload["chunks"]:
+                    for span in chunk["spans"]:
+                        if span["name"] == "dd.dynamic.span":
+                            span_hash[span["meta"]["debugger.probeid"]] = span
+                        else:
+                            for key, value in span["meta"].items():
+                                if key.startswith("_dd.di"):
+                                    span_hash[value] = span["meta"][key.split(".")[2]]
 
-    return hash
+    print(span_hash)
```

Comment:
Could you remove the print ?

Reply:
Fixed here de6a537028ac016bb10d972b0903b79c9c02bf26